Project 1 - Part 1/1 : Write 10 nouns and 3 actions that it performs.

AlexParadisProg opened the pull request:
Project 1 - Part 1 / 1 
First PFM C++ Homework. 

Added 10 nouns, with each noun having 3 functions attached to each of them.

AlexParadisProg committed:
Project 1 - Part 1/1 : Write 10 nouns and 3 actions that it performs.

matkatmusic commented on main.cpp:
```diff
@@ -52,84 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
-
-//  action 3:
- 
+//  Noun: Lion
+//  action 1: the lion sleeps
+lion.sleep();
+//  action 2: the lion walks
+lion.walk();
+//  action 3: the lion hunts
+lion.hunt();
```
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
**REPLACE** the highlighted noun/action as follows:
- [x] come up with a new noun and 3 actions
- [x] pick a noun that needs `camelCase` for the name
- [x] pick actions that need `camelCase` for their name.
**Do not** create an 11th noun.

AlexParadisProg committed:
New Commit to get familiar with Pull Request Review Process :
- Came up with new noun Electric Bike (for camelCase).
- Came up with 3 actions that need camelCase for their name.
